pytest-embedded-idf/pytest_embedded_idf/serial.py

```diff
@@ -5,7 +5,7 @@
 from typing import Optional, TextIO, Union
 
 import esptool
-from pytest_embedded.log import live_print_call
+from pytest_embedded.log import DuplicateStdoutPopen, live_print_call
 from pytest_embedded_serial_esp.serial import EspSerial, EsptoolArgs
```

Comment:
same, `LinuxSerial` does not require installing `pytest_embedded_serial_esp`.

I recommend moving all linux related classes into `linux.py`